src/_pytest/skipping.py

```diff
@@ -211,7 +211,25 @@ def show_simple(terminalreporter, lines, stat):
         for rep in failed:
             verbose_word = _get_report_str(config, rep)
             pos = _get_pos(config, rep)
-            lines.append("%s %s" % (verbose_word, pos))
+
+            line = "%s %s" % (verbose_word, pos)
+            try:
+                msg = rep.longrepr.reprcrash.message
+            except AttributeError:
+                pass
+            else:
+                # Only use the first line.
+                # Might be worth having a short_message property, which
+                # could default to this behavior.
+                i = msg.find("\n")
+                if i != -1:
+                    msg = msg[:i]
+                max_len = terminalreporter.writer.fullwidth - len(line) - 2
```

Comment:
For sake of legibility, I suggest:

```python
sep = ": "
max_len = terminalreporter.writer.fullwidth - len(line) - len(sep)
...
line += sep + msg
```

But you probably need to adjust for my comment to avoid using `…`, probably in favor of `...`.

Also can we move this code to a function and unittest it?

Reply:
Done - but needs to be revisited.